module/programs/helper.js

```diff
@@ -34,7 +34,7 @@ module.exports = class ProgramsHelper {
    * @returns {Object} - Programs list.
    */
 
-  static list(pageNo = '', pageSize = '', searchText, filter = {}, projection) {
+  static list(filter = {}, projection,pageNo = '', pageSize = '', searchText ) {
```

Comment:
@praveenKDass change in order of param won't affect places where this function being called?

Reply:
No ,verified and modified with other places where its used , The thing is in other helper files where ever this function called its passing as above params @VISHNUDAS-tunerlabs